{"name": "Test case ID: TS06", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "test_setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3b31de258cf01b29595a16b22e68004a\")>\n\n    @allure.description(\"validate navigation to the commercal\")\n    @allure.severity(severity_level=\"NORMAL\")\n    @allure.title(\"Test case ID: TS06\")\n    def test_5_Navigate_to_Commercial_man_thumbs_up_button(test_setup):\n        try:\n            driver = test_setup\n            hp = HomePage(driver)\n            lgn = Login(driver)\n            hp.click_save_button()\n            lgn.move_to_login()\n            lgn.login(\"0586691039\")\n            lgn.submit_login()\n            lgn.login_input_squares()\n            lgn.submit_secret_code()\n            hp.click_right_arrow()\n            hp.click_right_arrow()\n            hp.commercial_man_thumbs_up()\n            assert hp.assert_comercial_man_thumbs_up()\n        except Exception as e:\n            print(e)\n            driver.save_screenshot(\"test_TS06_homepage_error.png\")\n>           raise e\n\ntests\\test_homepage.py:93: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntests\\test_homepage.py:88: in test_5_Navigate_to_Commercial_man_thumbs_up_button\n    hp.commercial_man_thumbs_up()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Src.pages.HomePage.HomePage object at 0x000002355698D050>\n\n    def commercial_man_thumbs_up(self):\n        commercial_xpath = '//*[@id=\"root\"]/div/div[2]/div[2]/div/div/div[2]/div/div[2]/div[1]/div/div/div/div/ul/li[4]/a'\n        element = self.driver.find_element(By.XPATH, commercial_xpath)\n        self.driver.execute_script(\"arguments[0].click();\", element)\n>       self.driver.switch_to.window(self.driver.window_handles[1])\nE       IndexError: list index out of range\n\nSrc\\pages\\HomePage.py:81: IndexError"}, "description": "validate navigation to the commercal", "attachments": [{"name": "stdout", "source": "dbd7ff17-ed30-4c9e-badb-68538221e6ea-attachment.txt", "type": "text/plain"}], "start": 1680419599840, "stop": 1680419602240, "uuid": "e0db3e3d-fb8a-419e-9954-b75c5cff2590", "historyId": "f735f0d2c2f9cd9cec511b20257dd28e", "testCaseId": "a6307978e8260ebfacf122478b3c4b77", "fullName": "tests.test_homepage#test_5_Navigate_to_Commercial_man_thumbs_up_button", "labels": [{"name": "severity", "value": "NORMAL"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_homepage"}, {"name": "host", "value": "DESKTOP-9HU6DH7"}, {"name": "thread", "value": "14412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_homepage"}]}